gsa/src/web/pages/reports/downloadreportdialog.js

```diff
@@ -54,7 +55,9 @@ const DownloadReportDialog = ({
   includeOverrides = COMPOSER_CONTENT_DEFAULTS.includeOverrides,
   reportFormatId,
   reportFormats,
+  showThresholdMessage,
```

Comment:
Otherwise you should required the prop.
```suggestion
  showThresholdMessage = false,
```